### PR TITLE
test partition performance for index

### PR DESCRIPTION
**Pull request by boshengchen**

### CODE REVIEW

**Comment by bright-starry-sky** [comment]
I don't think this test needs to be added to the regression test. Because it runs too long, it will affect other tests.

**Comment by dangleptr** [comment]
Don't close it.  The perf is valuable,  add it into our repo,  of course we don't need to run it as UT

File: src/kvstore/test/CMakeLists.txt [hunk 1]
```diff
@@ -23,6 +23,13 @@ nebula_add_test(
     LIBRARIES ${THRIFT_LIBRARIES} ${ROCKSDB_LIBRARIES} wangle gtest
 )
 
+nebula_add_test(
```
Comment (dangleptr) [inline src/kvstore/test/CMakeLists.txt]: Use nebula_add_executable, we don't need to add it as a UT

**Comment by bright-starry-sky** [comment]
let me review it . I will open a new PR if necessary.

**Comment by bright-starry-sky** [comment]
Close this one. We will tracking it via #1807

**Pull request closed without merging**